### PR TITLE
Add note about external icon sets

### PR DESCRIPTION
**Pull request by tarekoraby**
Related to #721.

### CODE REVIEW

File: articles/components/icons/index.asciidoc [hunk 1]
```diff
@@ -81,6 +81,12 @@ return html`
 
 The icons are rendered as an inline `<svg>` element inside the shadow root of a `<vaadin-icon>` element, allowing you to style them using CSS.
 
+== External Icon Sets
+
+Other than the built-in Lumo and Vaadin icons, you can also use icons from other icon sets.
```
Comment (dougjewsbury) [inline articles/components/icons/index.asciidoc:86]:
```suggestion
Apart from the built-in Lumo and Vaadin icons, you can also use icons from other icon sets.
```

File: articles/components/icons/index.asciidoc [hunk 2]
```diff
@@ -81,6 +81,12 @@ return html`
...
+However, different icon sets tend to be implemented rather differently.
+This, unfortunately, implies that there is no standard, simple steps to follow that would work for all icon sets.
+Sill, as an example, you can consider how projects generated from https://start.vaadin.com[Vaadin Start] use the icons from the https://icons8.com/line-awesome/[Line Awesome] icon set.
```
Comment (dougjewsbury) [inline articles/components/icons/index.asciidoc:89]:
```suggestion
Unfortunately, this implies that there are no standard, simple steps to follow that would work for all icon sets.
Nevertheless, as an example, you can consider how projects generated from https://start.vaadin.com[Vaadin Start] use the icons from the https://icons8.com/line-awesome/[Line Awesome] icon set.
```

File: articles/components/icons/index.asciidoc [hunk 3]
```diff
@@ -81,6 +81,12 @@ return html`
 
 The icons are rendered as an inline `<svg>` element inside the shadow root of a `<vaadin-icon>` element, allowing you to style them using CSS.
 
+== External Icon Sets
+
+Apart from the built-in Lumo and Vaadin icons, you can also use icons from other icon sets.
+However, different icon sets tend to be implemented rather differently.
```
Comment (rolfsmeds) [inline articles/components/icons/index.asciidoc:87]:
```suggestion
However, icon sets come in many different formats, which means that there is no standard way to integrate and use them with Vaadin that would work for all icon sets.
```

File: articles/components/icons/index.asciidoc [hunk 4]
```diff
@@ -81,6 +81,12 @@ return html`
 
 The icons are rendered as an inline `<svg>` element inside the shadow root of a `<vaadin-icon>` element, allowing you to style them using CSS.
 
+== External Icon Sets
```
Comment (rolfsmeds) [inline articles/components/icons/index.asciidoc:84]:
```suggestion
== Third-party Icon Sets
```

**Review by rolfsmeds — CHANGES_REQUESTED** [review]

**Comment by jouni** [comment]
Added a similar note in #1612

**Comment by tarekoraby** [comment]
Closing in favor of https://github.com/vaadin/docs/pull/1612.

**Pull request closed without merging**